e2e/wdio/core/tests/alert.e2e-spec.ts

```diff
@@ -0,0 +1,92 @@
+import { AlertPo } from '../pages/alert.po';
+import {
+    click,
+    doesItExist,
+    getElementArrayLength,
+    getImageTagBrowserPlatform,
+    refreshPage,
+    saveElementScreenshot,
+    waitForElDisplayed,
+    waitForInvisibilityOf,
+    checkElementScreenshot, scrollIntoView
+} from '../../driver/wdio';
+import { checkElArrIsClickable } from '../../helper/assertion-helper';
+
+describe('alert test suite', function() {
+    const alertPage = new AlertPo();
+    const {
+        alerts,
+        closeAlertButton,
+        openOverlayButton,
+        popupAlert,
+        button,
+        openCustomAlertButton
+    } = alertPage;
+
+    beforeAll(() => {
+        alertPage.open();
+    }, 1);
+
+    afterEach(() => {
+        refreshPage();
+        waitForElDisplayed(alerts);
+    }, 1);
+
+    describe('main checks', function() {
+        it('should check ability to dismiss alert', () => {
+            checkElArrIsClickable(closeAlertButton);
+        });
+
+        it('should check popup alerts', () => {
+            const openOverlayButtonCount = 3;
+
+            checkPopupAlert(openOverlayButton, openOverlayButtonCount);
+        });
+
+        it('should check custom popup alerts', () => {
+            const customAlertCount = getElementArrayLength(openCustomAlertButton);
+
+            checkPopupAlert(openCustomAlertButton, customAlertCount);
+        });
+
+        it('should check RTL/LTR orientations', () => {
+            alertPage.checkRtlSwitch();
+        })
+    });
+
+    describe('visual regression', function() {
+        it('should check example blocks visual regression', () => {
+            alertPage.saveExampleBaselineScreenshot('alert');
+            expect(alertPage.compareWithBaseline('alert')).toBeLessThan(4);
+        });
+
+        it('should check custom alerts visual regression', () => {
+            const customAlertCount = getElementArrayLength(openCustomAlertButton);
+
```

Comment:
Same as above

Reply:
@nikvalor In general it is better to keep this space here. We have the guidelines here: https://www.notion.so/Automated-Tests-Styleguide-ad092cdea96d4d36a6fa24328688da43#347d5031a2d140e7b45ddd05e3b39205 
so in general, should separate by arrangement-action-assertion. I removed spacing for the smaller tests since having only 2 lines maybe doesn't make sense to add a space, but for this one I want to keep it. Makes it easier to read.

Reply:
Yeah, you are right